### ISSUE TITLE
Firefox addon can not install

### ISSUE THREAD

**Issue by massongit** [textlint/editor#61]
I can not install firefox addon because https://addons.mozilla.org/ja/firefox/addon/textlint-editor/ is 404.

**Comment by azu** [comment]
textlint editor for Firefox was rejected by following reason.

> Remotely executing code is not permitted in any form, including using Web Workers as a sandbox.

Current textlint editor require to install worker script as remtely executing code.
It is similar with Greasemonkey-like extensions, I think.
https://addons.mozilla.org/ja/firefox/addon/tampermonkey/
https://addons.mozilla.org/ja/firefox/addon/greasemonkey/
This extension show dialog before installing the worker script for avoiding malcious script.
Also, the script is exectuted in WebWorker.  the worker script can not access webextension APIs/ContentScript APIs.

But, Firefox disallow to use this approch because this is remotely executing code.

---

If we will re-publish textlint editor, we need to bundle textlint scripts into this extension.
It means that the extension user can not install own script.
Instead of it, the extension user select rule from built-in rules.

This approch is similar with 
https://chrome.google.com/webstore/detail/textlint-proofreader/hdongmdneapmhfblomidbafplpanpdmm?hl=ja
https://github.com/io-monad/textlint-chrome-extension

**Comment by massongit** [comment]
If you take a long time to resolve it, I think you can remove the Firefox add-on URL from `README.md` .

**Comment by azu** [comment]
Yes I did https://github.com/textlint/editor/commit/9bff34ef1ff04f054a4e8c160f95227f87fd1aca

**Comment by onokatio** [comment]
@azu  Could you distribute firefox .xpi file as Github release instead of AMO ?

**Comment by azu** [comment]
Firefox will require sign to install addon.

- [Signing and distribution overview | Firefox Extension Workshop](https://extensionworkshop.com/documentation/publish/signing-and-distribution-overview/#distributing-your-addon)

I do not understand that Self-distribution for public is allowed in Mozilla Addon Guideline.
